lms/djangoapps/grades/rest_api/v1/tests/test_gradebook_views.py

```diff
@@ -245,6 +245,40 @@ def test_course_grade_frozen(self):
             expected_data['grades_frozen'] = True
             assert expected_data == resp.data
 
+    def test_can_see_bulk_management_non_masters(self):
```

Comment:
I'd say patch here too, to set the course modes specifically. Or, do an assert on the result of get_course_enrollment_details